.env

```
@@ -12,9 +12,12 @@ DELETE_SINGLE_NOTIFICATION="notifications"
 MARK_NOTIFICATION_AS_READ_URL="notifications/mark-read"
 FORGET_PASSWORD_URL="auth/forgotPassword"
 RESET_PASSWORD_URL="auth/resetPassword"
+SPECIFIC-PRODUCT="products/673e1cd711599201718280fb"
+PRODUCT_CART="cart"
+
 
 NEXT_PUBLIC_API_BASE="https://flower.elevateegy.com/api/v1"
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The NEXT_PUBLIC_API_BASE key is duplicated

(DuplicatedKey)